systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlIsolatedST.java

```diff
@@ -151,6 +155,33 @@ void testCruiseControlTopicExclusion() {
         KafkaRebalanceUtils.waitForKafkaRebalanceCustomResourceState(CLUSTER_NAME, KafkaRebalanceState.Ready);
     }
 
+    @Test
+    void testCruiseControlReplicaMovementStrategy() {
+        List<String> actualStrategies;
+        List<String> newReplicaMovementStrategies = Arrays.asList("PrioritizeSmallReplicaMovementStrategy",
+                                                                    "PrioritizeLargeReplicaMovementStrategy",
+                                                                    "PostponeUrpReplicaMovementStrategy");
+
+        KafkaResource.kafkaWithCruiseControl(CLUSTER_NAME, 3, 3).done();
+        KafkaRebalanceResource.kafkaRebalance(CLUSTER_NAME).done();
+
+        actualStrategies = KafkaRebalanceResource.kafkaRebalanceClient().inNamespace(NAMESPACE)
```

Comment:
How about remove the `List<String> actualStrategies;` and do it like this:
```suggestion
        List<String> actualStrategies = KafkaRebalanceResource.kafkaRebalanceClient().inNamespace(NAMESPACE)
```

Reply:
Yeah, I forgot to update it, as originally I was assigning two values to `actualStrategies`